KaraIOke/Services/AppEnvironment/AppEnvironmentService.cs

```diff
@@ -15,16 +15,20 @@ public AppEnvironmentService(ISearchService searchMockService, ISearchService se
     }
 
     public ISearchService SearchService { get; private set; }
+    public IPlaylistService PlaylistService { get; private set; }
 
     public void updateDependencies(bool useMockServices)
     {
         if (useMockServices)
         {
             SearchService = _searchMockService;
+            PlaylistService = new PlaylistMockService();
         }
         else
         {
             SearchService = _searchService;
+            //TODO: implement PlaylistService
```

Comment:
I see we have a class for `PlaylistService` already, so we can assign it here, same flow as with `PlaylistMockService`, MauiProgram -> constructor -> private variable.